lib/fika/expressions.ex

```diff
@@ -49,7 +56,8 @@ defmodule Fika.Parser.Expressions do
   exp =
     choice([
       exp_match,
-      exp_add_op
+      exp_add_op,
+      exp_unary_op
```

Comment:
I think we don't need to add this here, because the parser would be able to parse unary ops by chaining these:

exp -> exp_add_op -> exp_bin_op -> exp_mult_op -> term -> factor -> non_literal_exps -> exp_unary_op